modules/cluster-logging-rn-5.5.3.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-3_{context}"]
+= Logging 5.5.3
+This release includes link:https://access.redhat.com/errata/RHBA-2022:6858[OpenShift Logging Bug Fix Release 5.5.3].
+
+[id="openshift-logging-5-5-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update log entries that had structured messages included the original message field which made the entry larger. This update removes the message field for structured logs to reduce the increased size. (link:https://issues.redhat.com/browse/LOG-2759[LOG-2759])
+
+* Before this update, the collector configuration excluded logs from `collector`, `default-log-store`, and `visualization`  pods, but was unable to exclude logs archived in a `.gz` file. With this update, archived logs stored as `.gz` files of `collector`, `default-log-store`, and `visualization` pods are also excluded. (link:https://issues.redhat.com/browse/LOG-2844[LOG-2844])
+
+* Before this update, when requests to an unavailable pod was sent through the gateway, no alert would fire to warn of the disruption. With this update, the individual alerts will fire if the gateway has issues completing a write or read request respectively. (link:https://issues.redhat.com/browse/LOG-2884[LOG-2884])
+
+* Before this update, pod metadata could be altered by fluent plugins because the values passed through the pipeline by reference. This update fixes that to ensure each log message receives a copy of the pod metadata so each message processes independently. (link:https://issues.redhat.com/browse/LOG-3046[LOG-3046])
+
+* Before this update, selecting 'unknown' severity in the OpenShift Console Logs view excluded logs with a `level=unknown` value. With this update, logs without level and with `level=unknown` values are visible when filtering by 'unknown' severity. (link:https://issues.redhat.com/browse/LOG-3062[LOG-3062])
+
+* Before this update, log records sent to Elasticsearch had an extra field named `write-index` which contains the name of the index to which the logs need to be sent. This field is not a part of the data model. After this update, this field is no longer sent. (link:https://issues.redhat.com/browse/LOG-3075[LOG-3075])
+
+* With the introduction of the new built-in link:https://cloud.redhat.com/blog/pod-security-admission-in-openshift-4.11[Pod Security Admission Controller] that enforces the Pod Security Standards, Pods that not configured in accordance with the enforced security standards defined globally or on the namespace level cannot run. With this update, the operator and collectors allow privileged execution and will run without security audit warnings or errors. (link:https://issues.redhat.com/browse/LOG-3077[LOG-3077])
```

Comment:
I had to read this a couple of times to understand it. Let me know if I am understanding it correctly and what you think of this rephrase:

`With the introduction of the new built-in link:https://cloud.redhat.com/blog/pod-security-admission-in-openshift-4.11[Pod Security Admission Controller], it enforces the Pod Security Standards. Security standards provide global definitions and namespace level definitions. Pods that are not configured in accordance with either of these definitions cannot run. With this update, the operator and collectors allow privileged execution and run without security audit warnings or errors.`